### PR TITLE
Adds winter coats for department heads

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

Ports winter coats for department heads from tg.
https://github.com/tgstation/tgstation/pull/56854

# Spriting

image

# Changelog

:cl:  
rscadd: adds winter coats for department heads
/:cl:

### CODE REVIEW

**Comment by KillerOrcaCora** [comment]
Where are the in-hand sprites?

**Comment by ynot01** [comment]
adorable

**Comment by Mqiib** [comment]
RD's coat needs to have some ridiculous name like "Research Director's Advanced External Thermal Insulator" because science

The item descriptions are top-tier though.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -575,6 +588,17 @@
 /obj/item/clothing/head/hooded/winterhood/security
 	icon_state = "winterhood_security"
 
+/obj/item/clothing/suit/hooded/wintercoat/security/hos
+	name = "head of security's winter coat"
+	desc = "A red, armour-padded winter coat, lovingly woven with a Kevlar interleave and reinforced with semi-ablative polymers and a silver azide fill material. The zipper tab looks like a tiny replica of Beepsky."
+	icon_state = "coathos"
+	armor = list(MELEE = 35, BULLET = 25, LASER = 40, ENERGY = 50, BOMB = 35, BIO = 0, FIRE = 0, ACID = 55)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 35, BULLET = 25, LASER = 40, ENERGY = 50, BOMB = 35, BIO = 0, RAD = 0, FIRE = 0, ACID = 55)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -586,6 +610,24 @@
 /obj/item/clothing/head/hooded/winterhood/medical
 	icon_state = "winterhood_medical"
 
+/obj/item/clothing/suit/hooded/wintercoat/medical/cmo
+	name = "chief medical officer's winter coat"
+	desc = "A winter coat in a vibrant shade of blue with a small silver caduceus instead of a plastic zipper tab. The normal liner is replaced with an exceptionally thick, soft layer of fur."
+	icon_state = "coatcmo"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 50, FIRE = 20, ACID = 30)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 50, RAD = 0, FIRE = 20, ACID = 30)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -586,6 +610,24 @@
...
+/obj/item/clothing/suit/hooded/wintercoat/medical/cmo/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/melee/classic_baton,
+	)
+
+/obj/item/clothing/head/hooded/winterhood/medical/cmo
+	desc = "A blue winter coat hood."
+	icon_state = "winterhood_cmo"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 50, FIRE = 20, ACID = 30)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 50, RAD = 0, FIRE = 20, ACID = 30)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -597,6 +639,24 @@
 /obj/item/clothing/head/hooded/winterhood/science
 	icon_state = "winterhood_science"
 
+/obj/item/clothing/suit/hooded/wintercoat/science/rd
+	name = "research director's winter coat"
+	desc = "A thick arctic winter coat with an outdated atomic model instead of a plastic zipper tab. Most in the know are heavily aware that Bohr's model of the atom was outdated by the time of the 1930s when the Heisenbergian and Schrodinger models were generally accepted for true. Nevertheless, we still see its use in anachronism, roleplaying, and, in this case, as a zipper tab. At least it should keep you warm on your ivory pillar."
+	icon_state = "coatrd"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 20, BIO = 0, FIRE = 30, ACID = 0)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 0)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 5]
```diff
@@ -597,6 +639,24 @@
...
+/obj/item/clothing/suit/hooded/wintercoat/science/rd/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/melee/classic_baton,
+	)
+
+/obj/item/clothing/head/hooded/winterhood/science/rd
+	desc = "A white winter coat hood. It smells faintly of hair gel."
+	icon_state = "winterhood_rd"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 20, BIO = 0, FIRE = 30, ACID = 0)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 0)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 6]
```diff
@@ -608,6 +668,24 @@
 /obj/item/clothing/head/hooded/winterhood/engineering
 	icon_state = "winterhood_engineer"
 
+/obj/item/clothing/suit/hooded/wintercoat/engineering/ce
+	name = "chief engineer's winter coat"
+	desc = "A white winter coat with reflective green and yellow stripes. Stuffed with asbestos, treated with fire retardant PBDE, lined with a micro thin sheet of lead foil and snugly fitted to your body's measurements. This baby's ready to save you from anything except the thyroid cancer and systemic fibrosis you'll get from wearing it. The zipper tab is a tiny golden wrench."
+	icon_state = "coatce"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 30, ACID = 10)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 30, FIRE = 30, ACID = 10)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 7]
```diff
@@ -608,6 +668,24 @@
...
+/obj/item/clothing/suit/hooded/wintercoat/engineering/ce/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/melee/classic_baton,
+	)
+
+/obj/item/clothing/head/hooded/winterhood/engineering/ce
+	desc = "A white winter coat hood. Feels surprisingly heavy. The tag says that it's not child safe."
+	icon_state = "winterhood_ce"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 30, ACID = 10)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 30, ACID = 10)
```

**Review by tattax — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/miscellaneous.dm [hunk 8]
```diff
@@ -608,6 +669,24 @@
...
+/obj/item/clothing/suit/hooded/wintercoat/engineering/ce/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/melee/classic_baton,
+	)
+
+/obj/item/clothing/head/hooded/winterhood/engineering/ce
+	desc = "A white winter coat hood. Feels surprisingly heavy. The tag says that it's not child safe."
+	icon_state = "winterhood_ce"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 15, FIRE = 30, ACID = 10)
```
Comment (tattax) [inline code/modules/clothing/suits/miscellaneous.dm:694]:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 15, FIRE = 30, ACID = 10)
```

tbh if you're gonna add armor to the CE hood add it to the normal engie hood aswell (same with the other ones)

**Comment by tattax** [comment]
can you link the original PR and author btw?

**Comment by SapphicOverload** [comment]
> can you link the original PR and author btw?

https://github.com/tgstation/tgstation/pull/56854

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> head of security coat with 50 energy armor

that value seems incorrect

**Review by KillerOrcaCora — CHANGES_REQUESTED** [review]
Winter coat themselves need some form of in-hand sprites done.

**Comment by SapphicOverload** [comment]
> Winter coat themselves need some form of in-hand sprites done.

done

**Review by JamieD1 — APPROVED** [review]

**Review by KillerOrcaCora — COMMENTED** [review]
Alright, this is fine now.

**Review by KillerOrcaCora — APPROVED** [review]

**Pull request merged**